homeassistant/components/bbb_gpio.py

```diff
@@ -63,7 +63,7 @@ def read_input(pin):
     """Read a value from a GPIO."""
     # pylint: disable=import-error,undefined-variable
     import Adafruit_BBIO.GPIO as GPIO
-    return GPIO.input(pin)
+    return True if GPIO.input(pin) is GPIO.HIGH else False
```

Comment:
Changed as suggested. My python is very basic so haven't find out that but I liked how clear to understand was that construct.